plugins/dynamix.vm.manager/templates/Custom.form.php

```diff
@@ -1254,7 +1264,8 @@ function resetForm() {
 	}
 
 	$('.advancedview').change(function () {
-		if ($(this).is(':checked')) {
+		if 
```

Comment:
Was this meant to be split onto 2 lines? 🤔

Reply:
Will correct that since another change is necessary for Bitlocker to work.
Thank you @OmgImAlexis for the heads up! :)